specs/policy/attributes/attributes.openapi.yaml

```diff
@@ -1024,6 +1038,7 @@ components:
         withKeyAccessGrants:
           type: boolean
           title: with_key_access_grants
+          description: Deprecated
```

Comment:
Do you want to include the recommended approach for this as well (like you did other places)?

Reply:
Same as the other thread, this kind of smells like https://virtru.atlassian.net/browse/DSPX-1424 .

Good flag and thank you for the question!  I'll pull this thread a bit early next week 🤔 🧵